src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
But if timer is daemon, some timeout task may not be executed, and so, some request may not have any response or exception.